tests/text/test_perplexity.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
```

Comment:
lets use the testing template, cc @stancld

Reply:
@bryant1410 Could you please write tests according you our template? In case we have no direct reference package and function to use, one good example can be found [here (EED tests)](https://github.com/PyTorchLightning/metrics/blob/master/tests/text/test_eed.py).

Definitely let me know should you need any help! :]